examples/multimodal/Vision_Fine_tuning_on_GPT4o_for_Visual_Question_Answering.ipynb

```diff
@@ -0,0 +1,1241 @@
ohGHWYvTCBdZaUtJYjXiFZkV++yBfN63MFLSb0975g0K2IesIhm9vdJ/hcg/ppsr4pFb4gefm9fZjYAu3L1/06+fr73C3B6bNPPxDofvvdj26986biimKHw5UisZw1c7FMplDOPGPVqspbLr3p3kfvyyrLcoYdh08e+eWnny1K04cvvl1dURwPR8ZWjwxGPS+88czV514za/a0ZZ980VnX+vPy73raOy4977Kfjm5s6uyyFwb3N+/a2Fe3Onh4/tIbwrx3opJcS0rnz5vCcbXKkKpKsfhQxhdZh5aUehxdXceP+H7d0PDRt0IGdn+hKRGJN3Ukd1I0cbLPH7cYtU5ft9Qqvebe66JJSBhG6P/VPECb0B2mdyQpCAQwgig4eE1COyJI8VRDNSL1iTDEOQdvXz7JyQW5Ye3lgDo4xzAtkuHfhiEgg1FF2gYXYFugH+gKRACMRDJjBhxcpCW6HqhXbC9CJuesh37QTDM5/hVBijUwMzgjmvEtbcA7eoBfgFbpDa5KBDhu4V8ok09aQtiEfXEXW8MnQ3OvOOK/fopf0SGUz3DixtEM1GNrkKfwAEKEZCHiWAwB3GD4AA5QPMPoiRwNi2VobqQrFgiryb9sKzwLCjkchWBtaIyVHSyjGe35qZgei2LVwBxZBPgVwXHcKPh1Rd9rAs3Q1mMxhI8lxwhOFoA47w3OOdmxZeMVF100e+ashpOtXc1NGVkFH338/qLTFmvgnePCxDOmTTxz+rGNh1d8/tOXP32d0teU2pdSJhTmqrNSommp2vQUIR0Ha3xCcrR5uCDqs+e1Y6YO22taDrbtbWsX2qyCKSno7U0OZ4bbYjTElaxr/drdv91780Nwju09TR3x+lvfvV4zTl5Rnn/8aKvOr4qlJhRGbdgb+HjlroevmF2UanK5vH7ZgGAydYd64ybdtz/seGn3DkuZUFQsu+T+aXq56eih4z9v3UmRiMsuu79Em5ujzF6y6LKxVVN1yTR3j1cVI/2gJu6PB7qDzmP+F+95/9dDP3zy4oph0zOnLZhx4zVX9vfYNm1Y39ISdLYJL7z2Q2q6ULNrkLXxhoU7712dkj+Y1evM86eXDSsZ8HZsP7hv/ZrOweSQJmEwAZ1SSC8usmgyGyN1XqcnpTBrIGJXqZVqk9oTCQZ8LqXKpNUShGVsacCbRmuqmBEMGSOCXG1ShCkpT71KSqEFfFRPFNAGVRbqrdYE5XcoMClJqtAZuLviXZb6huaa9tYJikxnMLhw0eknvnjzicefuvPOW1NTLfFkVB6WpaWkxILxa8+/Mk2X+sEr77d621yCO6c49+qrrlg4ab5RKo9S9VOG2miwKu17z72dmZqhVSnj3vBAsDfFZrrp6tukGTrXkTWHvZ3PbloWVuhc2ZR6zd10smFoYXpEY9Bn5pUvPr1xxc8NMsmoU+YEJVJiVfyY0gpKqo2p+rEzbd29HY11wbpdsSO1gi0k+DJM6QVyAaNYkxBoW3TBrKrywkH4QeiUSIl2hMbxy0GFgvgCuUFEPPaQLc8zDwYPMG4uvFPFR537OPiWNjSALgir4JlH/kJPyjOPtR7owEkQsqIlNP63YQhy4k7YB2ge4oGWIGDGE8kbCBAxheFFiIENY0hugbD/4gv4F9KFFHGoQV/FVyyGe8mkgRYdaINFoo3IcwEEBMSKjsh/wRD904Cx6BPCRokrXhEnxlf/4firDRPjRlYBAyLG09I/O4WkyQRI1wRsAzq8B5Cw8AmCjSI1NTykiEGMTnt6YyxuB30Qffkl/rSnGmA7RQxiYuwVkxR/GBozZzJbo0QDWfCnwFcTIQ7mCNaJ/SQKj9+bdLdgK7wVHBapILdt2wZifvvVso6GEw21jeVlQ55/9OnrbrjJkG6hQ6q94sJLxpyAN1g6uyq04adewZlRWThh5pT963evaVyr69BnClklmvKMZNa4vHGKgAwXvUhvJDueVaDIn1Q1PpQM2CO2uq6Gk8HOFZ98H5g1L6Ms7ZedK3Yf3TmycnxxcbYn7Kx3HH/gq1uV4+XFi/P9csokCr1ul1Ga0e+P5OYbVu7rGzes+8xpOaSGpFjo/r7uJz5/uLG9M3u0YuaSUmsp5G2J+GPKaGR+1aQh4yvef+q3G2ffO6/srLSQSRWUopghPELQR806QzQS2LR98/rdG++49+6w3medtsTeeXT7Dx2Hdn4777yhJUNLr7jpXG+EOshuwLijw1E8XtBpqFMQHDeuyqQ3kTMmmkhbv+Xohj/2DSbIyJIah1kVqphGpW2p7+23t8jU0tyKrI5dntZ9PerhKZEQthF5aqZ0oNsd6rdFPDCIRiFQqMiebjVWDHhjGmvM5u1NdtYOAllKnnbI8Ky0VIlC7ooG7VhI1JjXdTG3m5rUeq0xJbOoO5j4+Mdvpt3yQMzlVkrlF5xx3qtfvrL01aXDMstVgrLjZHuq2qoWFEFncPq4qdNmzvAI7kdffsKYa5g/c74hqcf8r9OQMWQwGFpCzg4eHbXBa3MGXZEpeVOvvvgaryfsjktaba4eiSyYa6yev1gwWWJhT8+ODYGykKo8K4Sfc4k58/IlvatW7Pf0jj77nBBOU/Iw5SSLlZRakEaKDXmF+cYJE/a/+ZnQ12LSFEaZvhbfj0ZBPrDk1Nu9dueJtsa9e3b29NjXrFlHnBZgwWOPElbUYGIUwgIjvvJ5caJm5cnHv5ErIiWK5M/DCRXzVia5IEAG7XAFZh+Zg/cuVE8zCHyQmP/u8RcZE6eK0MT8OBgb2IPSxJlxzkXe/yJbwTmDiTBES8YWkYtmxN3ibQTZk4uExtA5SaAhfu4FqmjA4mnMifgvt9PPX8DH5P+aj7gQGjAWI/7bdfGbsi8MxF2ofuF0ABf+xbGKbpkG0ENiAFyqSE3AxOiHADfEJRwRcW4CsGCLGF1EIoagASPyG+DASQIjMAguD2coWC1sWzBEjPjPMwG5ML2j5WFFhGgQtYxtkd7QZPMjIT+jBgKPAD5+VNRnqL3ZZLTdCK
```

Comment:
where does results_w_scores_4o come from

Reply:
I explained in the cookbook that I wouldn't run the code for 4o for brevity, since its the same as the existing evaluation code! Wanted to optimize for readability here!

Reply:
Got it, but we should remove the code here then right? Otherwise the cookbook won't be able to run

Reply:
"To fully understand the impact of fine-tuning, we also evaluated the same set of test questions using the **non-fine-tuned GPT-4o** model. While the code for this process is not shown here for brevity, it is identical to the evaluation method shown earlier (with the exception of switching the model parameter)."

Reply:
Removed this caveat, added in the code for 4o eval